templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+<div class="lune-page lune-page-product {{classes}}">
+  <div class="md-col-9 mx-auto {{classes}}">
+
+    {{#hero}}
+      <section class="lune-product-section lune-product-section-hero relative z1 flex">
+        <div class="lune-hero-content col-12 md-mt0 flex flex-column items-center justify-center">
+          <div class="lune-hero-title-wrapper">
+            <h1 class="lune-hero-title relative caps m0 px3 md-p0 z1 lune-anim-move-up {{#title_parts.1}}lune-title-has-mobile-hyphen{{/title_parts.1}}" aria-label="{{full_title}}">{{title_parts.0}}</h1>
```

Comment:
if the title is complex and used across you can pull it out as a partial.
i think this entire structure should be 
```html
<h* aria-label=""><span aria-hidden="true"></span><span></span><span></span></h*>
```